### PR TITLE
Added recurring Model Management SIG Call

### PR DESCRIPTION
**Pull request by karlschriek**

### CODE REVIEW

File: calendar.yaml [hunk 1]
```diff
@@ -259,3 +259,18 @@
...
+  name: Model Management SIG Call
+  date: 26/5/2020
+  time: 8:00am-9:00am
+  frequency: bi-weekly
+  video: https://meet.google.com/dxz-zthy-xjn
+  description:
+    - |
+      Notes & agenda: https://docs.google.com/document/d/1R2r1z5O4USpn3BW29-4blCoPueB3p2LAjvdKa97fT7U/edit?usp=sharing
+
+      Join at: https://zoom.us/j/799749911
```
Comment (jlewi) [inline calendar.yaml:273]: I don't think you will have access to this zoom call. Do you have a different way to host the meeting e.g. Google meets where you can host the meeting?

**Comment by jlewi** [comment]
@karlschriek Is there an issue tracking setting up the call; in addition to figuring out how you will host the call you will need to get

kubeflow/community#342 resolved in order to update the calendar

File: calendar.yaml (hunk 1 above)
Comment (karlschriek) [reply calendar.yaml:273]: Sorry that was a copy paste error. The correct link is under "video" above

**Comment by karlschriek** [comment]
> ... you will need to get #342 resolved in order to update the calendar

I understood #342 as a request to remove the bottleneck of just a few people currently being able to do this. Or is it impossible to update the calendar at the moment?

**Pull request merged**